spec/rspec/its_spec.rb

```diff
@@ -376,5 +378,42 @@ def terminator
         end
       end
     end
+
+    describe "#fits" do
+      it 'calls #its with :focus metadata' do
+        stub = Class.new do
+          include RSpec::Its
+        end.new
+        allow(stub).to receive(:its)
+
+        expect(stub).to receive(:its).with('attribute', :focus).and_yield
+        stub.fits('attribute') {}
+      end
+
+      context 'when used within RSpec describe' do
+        subject { `bin/rspec spec/support/files/fits_example.rb --tag focus` }
+
+        it { expect(subject).to include('1 example, 0 failures') }
+      end
+    end
+
+    describe "#xits" do
+      it 'calls #its with :skip metadata' do
+        stub = Class.new do
+          include RSpec::Its
+        end.new
+        allow(stub).to receive(:its)
+
+        expect(stub).to receive(:its).with('attribute', :skip).and_yield
+        stub.xits('attribute') {}
+      end
+
+      context 'when used within RSpec describe' do
+        subject { `bin/rspec spec/support/files/xits_example.rb` }
+
+        it { expect(subject).to include('2 examples, 0 failures, 1 pending') }
+      end
+    end
```

Comment:
Please take a look at how `rspec-core` tests example groups, this isn't a solution we would want to merge.

Reply:
Hey @JonRowe I've updated the specs. Let me know if they're okay

Reply:
I guess Jon meant something more like [this](https://github.com/rspec/rspec-core/blob/bbe35666734e36706e18b8fa320ed037f59ccd4a/spec/rspec/core/example_group_spec.rb#L1059).

Reply:
I couldn't copy the simplicity of that example exactly, because we're working with `it (example)` vs `fits (describe)` so they need to be handled differently, reflected in the updated spec. Not sure if this is as clear as what I had before but it works I guess 🙁